wiki/advantages-and-features.md

```diff
@@ -0,0 +1,185 @@
+data from each scraper against each other limiting the trust needed. Scrapers are 
+run by several trusted members of the community located around the world. 
+Read more about scrapers on the [Scraper Wiki Page](Scraper "wikilink")
+
+Further, Gridcoin has a voting system to help keep Gridcoin aligned with the
+community and have meaningful discussions. Voting helps reduce conflicts by
+not having unexpected changes and having a way to voice your opinion without 
+needing to fork and split the community.
+
+
+### Comparison to CureCoin
+---
+#### What's CureCoin?
+CureCoin is a cryptocurrency that aims to reward folding@home work
+
+
+#### Why's Gridcoin Different?
+
+* CureCoin only rewards folding@home --- Gridcoin rewards many projects   
+    * CureCoin is designed only for folding@home
+        * Only one project can be rewarded
+        * Expansion requires rework of the system
+
+    * Gridcoin is designed for BOINC projects
+        * Rewards many existing projects and allows for rewarding newly created BOINC projects
+        * More rewarded areas of study and more scientists running and starting projects. 
+
+^
+^
+
+* CureCoin involves Proof of Work ---  Gridcoin uses Proof of Stake
+    * More energy used in Proof of Work
+        * Less used energy on scientific work in CureCoin
+    
+    * With Gridcoin more energy goes directly to science
+        * Less energy used by Proof of Stake
+
+^
+^
+
+* Gridcoin has more active development and community
+    * Comparison of development activity in the last month:
+        * [Gridcoin's here](https://github.com/gridcoin-community/Gridcoin-Research/pulse/monthly) 
+        * [CureCoin's here](https://github.com/cygnusxi/CurecoinSource/pulse/monthly)
+ 
+
+### Comparison to OByte 
+---
+#### What's OByte?
+Obyte (formally byteball) is a cryptocurrency that focuses on various different 
+aspects such as smart contracts. Among other bots, it has a bot that distributes out 
+OBytes for work on [World Community Grid](https://www.worldcommunitygrid.org)
+  
+(note you can earn both Gridcoin and OBytes at the same time)
+
+
+#### Why's Gridcoin Different?
+
+* OByte's main focus is different from Gridcoin 
+    * Science aspect is a secondary or tertiary goal
+        * Fewer OBytes sent to reward science and more goes to other distribution methods
+    
+    * For Gridcoin science is a primary goal
+        * Larger percentage rewards science 
+
+^
+^
+
+* OByte's bots are heavily centralized 
+    * Happening through one bot that could stop at any point
+        * Funding to this bot comes entirely through a grant from a fund 
+          controlled currently by 4 people
+
+    * In Gridcoin, coins are distributed as new coins in the network not through a single fund or bot
+
+^
+^
+
+* OByte only rewards World Community Grid and no other BOINC project
+    * On any other project, you can't earn OBytes for it
+
+    * With Gridcoin there are 18 project you can contribute to (as of 2020-08-08)
```

Comment:
This is fine as long as we make a note to update this whenever the whitelist is changed

Reply:
Hmm actually I am able to pull directly from the whitelist.yml file so it would only need to be updated in one place with  `{{ site.data.whitelist.projects.size }}`. I'm going to do a PR real quick to update the whitelist and then add that I think